app/models/language_model.rb

```diff
@@ -0,0 +1,30 @@
+# We don't care about large or not
+class LanguageModel < ApplicationRecord
+  def readonly?
+    !new_record?
+  end
+
+  def before_destroy
+    raise ActiveRecord::ReadOnlyRecord
+  end
```

Comment:
Based on my quick test on console, I think we can remove this before_destroy. It appears that the readonly? hack is sufficient to prevent destroy. Do you think we need to keep this for some reason?